addon/components/bourbon-select-field.js

```diff
@@ -84,7 +84,7 @@ export default Component.extend(SelectMixin, {
   actions: {
     updateSelection(selectedIndex) {
       if (selectedIndex === undefined) {
-        if (this.findValueObject(this.get('value'))) {
+        if (isPresent(this.findValueObject(this.get('value')))) {
```

Comment:
was false for `0` causing the prompt to show instead of the number.  added check in other places as well.

Reply:
Ah, yeah, classic JS.